marimo/_runtime/virtual_file.py

```diff
@@ -34,15 +34,22 @@ class VirtualFile:
     filename: str
     buffer: bytes
 
-    def __init__(self, filename: str, buffer: bytes) -> None:
+    def __init__(self, filename: str, url: string, buffer: bytes) -> None:
         self.filename = filename
-        self.url = f"/@file/{filename}"
+        self.url = url
         self.buffer = buffer
 
 
+EMPTY_VIRTUAL_FILE = VirtualFile(
+    filename="empty.txt",
+    url="/@file/empty.txt",
+    buffer=b"",
+)
```

Comment:
Do you need to pre-load this into the registry?

Reply:
i can - it never actually gets called from mo.download, but in future it may